infrastructure/migrations/0018_auto_20210928_1642.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2021-09-28 14:42
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.core.exceptions import ObjectDoesNotExist
+
+
+def populate_implementation_year(apps, schema_editor):
+    project = apps.get_model("infrastructure", "Project")
+    year = apps.get_model("infrastructure", "FinancialYear")
+
+    db_alias = schema_editor.connection.alias
+
+    try:
+        implementation_year_id = year.objects.using(db_alias).get(budget_year="2019/2020")
+    except ObjectDoesNotExist:
+        implementation_year_id = 1
```

Comment:
we should be able to merge master into this branch and this should change to the fixed implementation, right?

Reply:
Yup, merged